src/Test/Utilities/Desktop/CodeRuntime/DesktopRuntimeEnvironment.cs

```diff
@@ -377,10 +371,20 @@ CompilationTestData IInternalRuntimeEnvironment.GetCompilationTestData()
 
         private static readonly object s_consoleGuard = new object();
 
-        internal static void Capture(Action action, int expectedLength, out string output, out string errorOutput)
+        internal static void Capture(Action action, int? expectedLength, out string output, out string errorOutput)
```

Comment:
I don't think we need the `int?`. The comparison doesn't happen until the top level and that helper can use `expectedOutput == null` as a sentinel.